src/black/__init__.py

```diff
@@ -5382,6 +5380,8 @@ def normalize_invisible_parens(node: Node, parens_after: Set[str]) -> None:
                     node.insert_child(index, Leaf(token.LPAR, ""))
                     node.append_child(Leaf(token.RPAR, ""))
                 break
+            elif node.type in [syms.assert_stmt, syms.if_stmt]:
+                pass
```

Comment:
Further understanding required here; this `if...pass` path works 'somehow' because it prevents the introduction of unnecessary parentheses.

Open question: are those parentheses being added by the [RHS string-split bug](https://github.com/psf/black/pull/1958/files#r574026249)?  And how does that relate to the [`wrap_in_parentheses`](https://github.com/psf/black/pull/1958/files#diff-d23a6917d7262ca9815093bffb7148299e516575251959ed9b405b648e76aa29R5387) method call?

Reply:
If we're going to continue to use a specific list of symbols during this conditional check, then I think the relevant Python grammar rule may be 'small statements' ([ref](https://docs.python.org/3/reference/grammar.html) - `small_stmt` rule).

Reply:
This special case behaviour has been removed.